.devcontainer/devcontainer.json

```
@@ -1,18 +1,16 @@
 {
     "name": "AVD Development",
-    "mounts": [
-        "source=${localWorkspaceFolder}/ansible_collections/arista/avd,target=/home/avd/.ansible/collections/ansible_collections/arista/avd,type=bind"
-    ],
     "image": "ghcr.io/aristanetworks/avd/dev:python3.11",
     // The AVD collection can also be installed from any git branch.
-    // Comment mount section if this is the case.
+    // Uncomment the section below if it's required
     //
     // "containerEnv": {
     //     "AVD_GITHUB_REPO": "aristanetworks/avd",
     //     "AVD_BRANCH_NAME": "devel"
     // },
     // Run entrypoint script manually as it's ignored by dev container CLI otherwise.
     // The dev entrypoint is used to install Ansible collections and requirements, as they are not included with the dev version.
-    // "true" is required to exit "onCreateCommand" without entering ZSH.
-    "onCreateCommand": "/bin/entrypoint.sh true"
+    // "true" is required to exit "postCreateCommand" without entering ZSH.
```

Should it look like this?
```suggestion
    // "true" is required to exit "postStartCommand" without entering ZSH.
```